aria-practices.html

```diff
@@ -5927,6 +5927,697 @@ <h5>Intentional Key Conflicts</h5>
+        When the <code>aria-pressed</code>attribute is present and has a non-empty value,
+        the button is a toggle button.
+        A toggle button is a two-state button, either pressed or unpressed.
+      </p>
+
+      <p>
+        The <code>true</code> value indicates that the button is pressed, and
+        the <code>false</code>value indicates that the button is unpressed.
+      </p>
+
+      <p>
+        HTML does not have a native way to represent toggle buttons,
+        but <code>aria-pressed</code> can be used with the HTML <code>button</code> element.
+      </p>
+
+      <pre><code class="lang-html">&lt;button aria-pressed="false">Mute&lt;/button></code></pre>
+
+      <p>
+        See the <a href="#button">Button</a> design pattern for examples.
+      </p>
+    </section>
+
+
+    <section id="communicating_widget_states_aria-expanded">
+      <h3><code>aria-expanded</code></h3>
+
+      <p>
+        The <code>aria-expanded</code> attribute informs assistive technology users that an interactive element can be expanded,
+        meaning it owns or controls another element that can be collapsed or expanded.
+      </p>
+
+      <p>
+        For example, a button can toggle the visibility of some content, as in an accordion.
+      </p>
+
+      <pre><code class="lang-html">&lt;h2>&lt;button aria-controls="spoiler" aria-expanded="false">Spoil the end&lt;/button>&lt;/h2>
+      &lt;div id="spoiler">
+       &lt;p>Ultimately, all stars burn out and the black holes evaporate from Hawking radiation,
+       and the universe reaches maximum entropy.&lt;/p>
+      &lt;/div></code></pre>
+
+      <p>Clicking the button shows or hides the "spoiler" element with JavaScript,
+        and also changes the value of <code>aria-expanded</code> to <code>true</code> when showing,
+        and <code>false</code> when hiding.
+        Users of assistive technology would know from focusing the button whether it is collapsed or expanded.
+      </p>
+
+      <p>
+        The <code>aria-expanded</code> state is used in these design patterns:
+      </p>
+
+      <table>
+        <thead>
+          <tr>
+            <th>Design pattern</th>
+            <th>Description</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <a href="#accordion">Accordion</a>
+            </td>
+            <td>A section With show/hide functionality</td>
+          </tr>
+          <tr>
+            <td>
+              <a href="#combobox">Combobox</a>
+            </td>
+            <td>An input widget with an associated popup</td>
+          </tr>
+          <tr>
+            <td>
+              <a href="#disclosure">Disclosure</a>
+            </td>
+            <td>A button that controls visibility of a section of content</td>
+          </tr>
+          <tr>
+            <td>
+              <a href="#listbox">Listbox</a>
```

Comment:
The link for the listbox pattern has a capital L, so this link (#listbox) is broken.
However, I think that the id should be changed to lowercase instead of this link, for consistency with other pattern ids and the listbox role name.